src/Backend/Core/Layout/Sass/screen.scss

```diff
@@ -1,8 +1,8 @@
 @charset 'UTF-8';
 
 // Font Awesome
-$fa-font-path: '../../../../../bower_components/fontawesome/fonts';
-@import '../../../../../bower_components/fontawesome/scss/font-awesome';
+$fa-font-path: '/fonts';
```

Comment:
Uhu, but in that case the name "_bootstrap-variables.scss" isn't correct anymore